ui/components/Button.tsx

```diff
@@ -11,4 +11,24 @@ function Button(props: Props) {
   return <MaterialButton {...props} />;
 }
 
-export default styled(Button)``;
+export default styled(Button)`
+  &.applications {
```

Comment:
This seems a little backwards. As I understand this, when the `.applications` class is applied to the `Button`, it changes the border properties.

I feel like this should be inverted. The `Applications` page or some page that uses a `<Button />` should apply some overrides. This is a little un-semantic as is.

Maybe if we has a class that was `.square-border` or something, it would be closer to what we are trying to do.

Reply:
Agreed - looking at the Figma doc I'm only seeing buttons that look like this one - maybe we want to make it the default?

Reply:
> Agreed - looking at the Figma doc I'm only seeing buttons that look like this one - maybe we want to make it the default?

Maybe, but that is a bit out of scope for this PR. This highlights the need for a component library (the project we have been talking about).